erfa/ufunc.c.templ

```diff
@@ -10,7 +10,6 @@
  */
 
 #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
-#define Py_LIMITED_API 0x030900f0
```

Comment:
Would it make sense to leave a comment here that we use the limited API but by default set it in`setup.py`?

Reply:
wouldn't hurt, I'll add it now